release.sbt

```diff
@@ -33,11 +33,19 @@ generatePackageJson := s"""{
                           |  "name": "${name.value}",
                           |  "version": "${version.value}",
                           |  "description": "${description.value}",
-                          |  "main": "core/target/js-2.13/weapon-regex-opt.js",
+                          |  "main": "core/target/js-2.13/weapon-regex-opt/main.js",
                           |  "repository": {
                           |    "type": "git",
                           |    "url": "${homepage.value.get}"
                           |  },
+                          |  "keywords": [
+                          |    "regex",
+                          |    "regexp",
+                          |    "regular expression",
```

Comment:
Not sure if npm supports two words in a single keyword. Seems like another possibility for the pipeline to fail

Reply:
I think npm does support that. Have a look [here](https://nodesource.com/blog/the-basics-of-package-json-in-node-js-and-npm#thekeywordsproperty).

Reply:
Great!